src/ui/server/controllers/insurance/declarations/modern-slavery/index.ts

```diff
@@ -86,3 +96,40 @@ export const get = (req: Request, res: Response) => {
     userName: getUserNameFromSession(req.session.user),
   });
 };
+
+/**
+ * post
+ * Check Declarations - Modern slavery validation errors and if successful, redirect to the next part of the flow.
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.redirect} Next part of the flow or error page
+ */
+export const post = async (req: Request, res: Response) => {
```

Comment:
do we need a try catch? or later?

Reply:
Later as part of data saving - there's no async calls here (removed `async`).